docfx.json

```diff
@@ -126,6 +126,12 @@
             "featureFlags": [
                 "enable_uhf_ppe"
             ],
+            "contributorGuideUrl": "https://learn.microsoft.com/contribute/content/dotnet/dotnet-contribute",
```

Comment:
Please add the `open_source_feedback_` prefix to the fields.

open_source_feedback_contributorGuideUrl
open_source_feedback_issueTemplateName
open_source_feedback_productName
open_source_feedback_productDescription
open_source_feedback_productLogoLightUrl
open_source_feedback_productLogoDarkUrl

Reply:
Thanks @homing1 I mis-read your previous comment.

In the next commit, I'll also move this to the file meta data section.